### ISSUE TITLE
Biological data - UCL Centre for Clinical Microbiology 

### ISSUE THREAD

**Issue by fidiris** [OpenSourceTB/OSTB_Series_3#4]
A set of [12 compounds ](https://github.com/OpenSourceTB/OSTB_Series_3/issues/2#issuecomment-517242575)were sent to collaborators at the UCL Centre for Clinical Microbiology back in 2019. The MIC of 9 targets (see below) were evaluated against drug sensitive (DS) and drug resistant (DR) Mycobacterium tuberculosis (Mtb) clinical isolates and the reference laboratory strain H37Rv.

DS, DR and H37Rv biofilms were also prepared as models of a non-replicating (NR) state and used for MIC testing of the 9 compounds.

The biological data is now summarised in the [wiki](https://github.com/OpenSourceTB/OSTB_Series_3/wiki/Second-Set-of-Compounds) and the original report from Priya Solanki in Tim McHugh's group can be found [here](https://github.com/OpenSourceTB/OSTB_Series_3/blob/master/Biological%20Data/MIC%20testing%20of%20OSTB%20compounds%20-%20March%202020.pdf). 

In short, resistance was observed with MIC values at > 40 µM.

![Summary Scheme](https://user-images.githubusercontent.com/48478874/78019969-78380c00-7348-11ea-966b-7dd4d7e5662b.png)

It's important to note that the NR assay carried out at UCL is different from the original assay done at Cornell and GSK in 2015. The original assay involved rendering virulent Mtb H37Rv NR by simultaneous exposure to four host relevant stresses (Hypoxia, mild acid, reactive nitrogen species and restrictions to butyrate). See [ACS Infect. Dis. 2015, **1**, 580-585](https://pubs.acs.org/doi/abs/10.1021/acsinfecdis.5b00025) for a detailed description of the NR assay.
